packages/loctool/lib/Project.js

```diff
@@ -720,7 +723,8 @@ Project.prototype.close = function(cb) {
     if(!this.localizeOnly) {
         var dir = this.xliffsOut;
         var base = this.options.id;
-        var extractedPath = path.join(dir, base + "-extracted.xliff");
+        var fileFormat = this.settings.intermediateFormat || "xliff";
+        var extractedPath = path.join(dir, base + "-extracted." + fileFormat);
```

Comment:
Extension string should be separate from the format identifier

Reply:
Right now, they are exactly the same, so it is a little silly to introduce a mapping right now.